Apps/Contoso.Android.Puppet/Contoso.Android.Puppet.csproj

```diff
@@ -59,6 +59,9 @@
     <AndroidLinkSkip>Xamarin.Android.Support.v7.AppCompat;Xamarin.Android.Support.Design</AndroidLinkSkip>
     <AndroidLinkTool>r8</AndroidLinkTool>
   </PropertyGroup>
+  <PropertyGroup>
+    <AndroidResgenExtraArgs>-G aapt_rules.txt $(AndroidResgenExtraArgs)</AndroidResgenExtraArgs>
```

Comment:
What was the issue?

Reply:
Crash on opening any dialog, was temporary fixed by updating Xamarin.Android